code/modules/surgery/organs/internal/cyberimp/augments_internal.dm

```diff
@@ -151,8 +151,8 @@
 	sparks.set_up(5, 1, src)
 	sparks.start()
 
-	owner.add_traits(list(TRAIT_IGNOREDAMAGESLOWDOWN, TRAIT_BATON_RESISTANCE, TRAIT_STUNIMMUNE), REF(src))
-	addtimer(TRAIT_CALLBACK_REMOVE(owner, TRAIT_IGNOREDAMAGESLOWDOWN, REF(src)), stun_resistance_time)
+	ADD_TRAIT(owner, TRAIT_STUNIMMUNE, REF(src))
+	ADD_TRAIT(owner, TRAIT_BATON_RESISTANCE, REF(src))
```

Comment:
It seems you're removing the damage slowdown immunity from the anti-stun implant buff. That's a tad lame.

Reply:
no that's just a mistake